packages/ui/src/proposals/model/proposalDetails.ts

```diff
@@ -1,5 +1,5 @@
-import { lowerFirstLetter } from '@/common/helpers'
-import { ProposalDetails } from '@/proposals/types'
+import { lowerFirstLetter } from '../../common/helpers'
+import { ProposalDetails } from '../types'
```

Comment:
Could you check you settings? This change looks like unnecessary

Reply:
Well, we use `proposalDetails.ts` and `proposalStatus.ts` in mock generators, and it couldn't find file when `@/...` is used. Maybe You have solution for this?